apps/server/src/services/integration-service/OscIntegration.ts

```diff
@@ -113,6 +110,44 @@ export class OscIntegration implements IIntegration<OscSubscription, OSCSettings
     }
   }
 
+  private stringToOSCArgs(argsString): ArgumentType[] {
+
+    // NOTE: regex taken from https://stackoverflow.com/questions/4031900/split-a-string-by-whitespace-keeping-quoted-segments-allowing-escaped-quotes
+    const parseRegex = /[\w.]+|"(?:\\"|[^"])+"/g;
+
+    const parsedArguments = argsString.match(parseRegex).map((argString: string) => {
```

Comment:
suggestion: would it be easier to extract the matching? 
I wonder if that would make the logic easier to leverage, we could also bail out early if there are no matches (null)

something like (untested):

```ts
  const matches = argsString.match(parseRegex);

  if (!matches) {
    return argsString 
  }
```